marketplace/commercetools-products/src/field.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Spinner } from '@contentful/forma-36-react-components';
+import { ApolloProvider } from '@apollo/react-components';
+import { SortableContainer } from 'react-sortable-hoc';
+import arrayMove from 'array-move';
+
+import { getApolloClient } from './apollo.js';
+import { SortableProductCard } from './product-card.js';
+
+const SortableList = SortableContainer(function SortableList({
+  items,
+  removeItem,
+  locale,
+  sortable,
+  isDisabled,
+  projectKey
+}) {
+  return (
+    <ol className="product-list">
+      {items.map((sku, index) => {
+        return (
+          <li key={index} className="product-list__item">
```

Comment:
Use something more stable (SKU?)

Reply:
You're using CSS class here. I'd either stick to CSS everywhere or convert to CSS-in-JS everywhere.